cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -129,6 +129,9 @@ features:
       updated-label:
         plural: seconds ago
         singular: second ago
+      primaryLabelOne: DAY
```

Comment:
talked to @elfenheart about this. Will take it up in subsequent cleanup of admin view in the subsequent release.